packages/wallet/src/redux/funding/indirect-funding/player-a/state.ts

```diff
@@ -0,0 +1,76 @@
+import { Properties } from '../../../utils';
```

Comment:
Commented on Andrew's PR as well about this, but we should be careful about directory and file name convention. In other parts of our code, we are currently using camel case like `channelState` as opposed to dashes like `indirect-funding`